skyxplore-ui-test/src/test/java/com/mentoring/pageobject/FactoryPage.java

```diff
@@ -64,15 +67,29 @@ public String getQuantityInQueue() {
         return quantityInQueue.getText();
     }
 
-    @Step("Checking the production queue process started.")
-    public boolean isQueueProcessStarted() {
+    @Step("Checking the queue process bar is visible.")
+    public boolean isQueueProcessBarVisible() {
         try {
-            log.debug("Waiting for production queue to start");
+            log.debug("Waiting for production queue job");
             Awaitility.await().pollInterval(500, TimeUnit.MILLISECONDS)
                     .atMost(15, TimeUnit.SECONDS)
                     .until(() -> getText(queueProcessBar).contains("00:"));
         } catch(ConditionTimeoutException e) {
-            log.warn("Production queue not started");
+            log.warn("Queue process bar is not visible");
+            return false;
+        }
+        return true;
+    }
+
+    @Step("Checking the queue is not visible anymore.")
+    public boolean isQueueInvisible() {
+        try {
+            log.debug("Waiting for production queue job");
+            Awaitility.await().pollInterval(500, TimeUnit.MILLISECONDS)
+                    .atMost(15, TimeUnit.SECONDS)
+                    .until(() -> (!isElementDisplayedNoWait(queueElement)));
+        } catch(ConditionTimeoutException e) {
+            log.warn("Queue is not disappeared");
```

Comment:
Minor: Has not disappeared or did not disappear.

Reply:
Fixed, thx. Also modified the method name: isQueueInvisible ->isQueueDisappeared (it is more understandable for me).

Reply:
Is it OK? :-)